cmd/auth.go

```diff
@@ -31,7 +32,8 @@ func newLoginCommand(astroClient astro.Client, out io.Writer) *cobra.Command {
 			return login(cmd, args, astroClient, out)
 		},
 	}
-	cmd.Flags().BoolVarP(&loginLink, "login-link", "l", false, "Get login link to login on a separate device for browserless cloud CLI login")
+	cmd.Flags().BoolVarP(&loginLink, "login-link", "l", false, "Get login link to login on a separate device for cloud CLI login")
```

Comment:
For readability, I think we should rename the variable loginLink to shouldDisplayLoginLink or something related to the behaviour/use of this variable.

Reply:
I updated this